client/common/directives/ModelFieldPointsOfInterest/ModelFieldPointsOfInterest.js

```diff
@@ -114,14 +111,18 @@ angular.module('dashboard.directives.ModelFieldPointsOfInterest', [
 				scope.searchResults = [];              // All data recieved from query
 				scope.displayedMarkers = [];           // Markers that match query
 				scope.displayedSearchResults = [];     // Data for location list
+				scope.displayOptions = scope.property.display.options;
 				scope.isMapLoading = true;
 				scope.isLoaded = false;
 				scope.placeType = scope.property.display.options.placeType; //Default query value
 				scope.googleApiKey = scope.property.display.options.googleApiKey;
 				scope.googleType = [convertStringToGoogleTypeFormat(scope.placeType)];
+				scope.initalLoad = true;
+				scope.$watch('data.phoneNumber', function(newVal, oldVal) {
+					if (!newVal) scope.data.phoneNumber = oldVal;
```

Comment:
Why is this necessary?

Reply:
This was added to prevent phonenumber to be null. Removed on the update!